botocore/docs/bcdoc/restdoc.py

```diff
@@ -206,7 +211,10 @@ def flush_structure(self):
         value = self.getvalue()
         for name, section in self._structure.items():
             value += section.flush_structure()
-        return value
+        # Ignores response/request sections if the line number exceeds our limit.
+        line_count = value.decode('utf-8').count('\n')
```

Comment:
Would `len(value.decode('utf-8').splitlines())` be a more accurate measure since that method splits on other newline characters besides `\n`?

Reply:
After running the doc generation again with this change it only added 1 extra line to the count for each section. I'll add this change to future proof incase we ever have other newline characters to catch. Thanks!